packages/@lwc/jest-preset/package.json

```
@@ -22,6 +22,7 @@
     ],
     "peerDependencies": {
         "@lwc/compiler": "*",
+        "@lwc/engine": "*",
         "@lwc/synthetic-shadow": "*",
```

Should we specify `lwc` here instead of individual `@lwc/*` packages? We tell users to add `lwc` as a dependency instead of individual `@lwc/*` packages in their projects, but these users see installation warnings when they install our `lwc-test` packages because our `lwc-test` packages list out `@lwc/*` packages as peer dependencies.

```suggestion
        "lwc": "*",
```